### ISSUE TITLE
Line Protocol ingestion with influxSkipSingleField for one field line without measurement

### ISSUE THREAD

**Issue by johnatannvmd** [VictoriaMetrics/VictoriaMetrics#1139]
**Describe the bug**
According to the documentation, this option should skip field names and use the only measurement for the metric names.
```
-influxSkipSingleField
    	Uses '{measurement}' instead of '{measurement}{separator}{field_name}' for metic name if Influx line contains only a single field
```
Also according this commit 1595dcd3d9b42fae03f1605b053fb24fae55da70 we are able to send lines like this(without measurement):
`,tag=name field=1 123456789`
and it should perfectly be saved in the VM as-is. But this does not work with `influxSkipSingleField`.

This if statement prevents to use of one field without a measurement(I refer to cluster version but single node should be affected as well):
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/cluster/app/vminsert/influx/request_handler.go#L145

To make things work I change that line with:
```go
if !skipFieldKey || r.Measurement == "" {
```
but that looks like a hack and I want to know if this behavior(influxSkipSingleField + single filed without measurement) is correct or not.

Want to achieve this behavior(`_` - at the beginning of the line is the space symbol `␣`):
![image](https://user-images.githubusercontent.com/4918448/111909006-8a84eb80-8a6c-11eb-8975-c8f10cc29ba7.png)

**To Reproduce**
Send to the VM with `influxSkipSingleField` enabled
`,tag=influxSkipSingleField field=1`
Got no saved points

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./vmstorage --version
vmstorage-20210303-100156-tags-v1.55.1-cluster-0-g0b8d9350d
```

**Used command-line flags**
`-influxSkipSingleField` - for vminsert, other params as is

**Test behavior**
I use a little script for testing, see below.

***Actual behavior***
`field{tag=influxSkipSingleField}` - no series
![image](https://user-images.githubusercontent.com/4918448/111908375-12b5c180-8a6a-11eb-86d1-7dbafd90532d.png)

***Expected behavior***
`field{tag=influxSkipSingleField}` - should appears
![image](https://user-images.githubusercontent.com/4918448/111908308-c79bae80-8a69-11eb-8237-ac2fdac2cc40.png)

***Little script that emulates ingestion in various crazy formats***
```python
from time import time
import requests

vm = 'http://localhost:8480'
tag = 'influxSkipMeasurement'

now = int(time())

for i in range(10):
    r = requests.post(vm + '/insert/0/influx/write?precision=s', data=f',tag={tag} field={i + 1} {now - (i * 10)}')
    if r.status_code != 204:
        print(r.text)
        exit(1)

    r = requests.post(vm + '/insert/0/influx/write?precision=s', data=f',tag={tag} field_1={i},field_2={10 - i} {now - (i * 10)}')
    if r.status_code != 204:
        print(r.text)
        exit(1)

    r = requests.post(vm + '/insert/0/influx/write?precision=s', data=f'measurement,tag={tag} data={i + 5} {now - (i * 10)}')
    if r.status_code != 204:
        print(r.text)
        exit(1)

    r = requests.post(vm + '/insert/0/influx/write?precision=s', data=f'measurement,tag={tag} data_1={i + 5},data_2={10 - i + 5} {now - (i * 10)}')
    if r.status_code != 204:
        print(r.text)
        exit(1)
```

**Comment by valyala** [comment]
@johnatannvmd , thanks for the detailed issue! It should be implemented in the commit 6ebf18ab9765548a20c2eb0ddfbb31a8fc4e8390 , which will be included in the next release of VictoriaMetrics.

**Comment by valyala** [comment]
VictoriaMetrics should use field name as metric name if `-influxSkipSingleField` is set and the measurement is empty starting from [v1.57.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing this feature request as done.
